plugin_example/README.md

```diff
@@ -0,0 +1,5 @@
+## DUNE plugins
+
+This directory contains DUNE example plugins. To test the example plugins, copy the contents of the [../plugin_example/](../plugin_example) directory into the [../src/plugin/](../src/plugin/) directory. This way, DUNE will automatically discover the new plugins.
```

Comment:
I presume symbolic links work, maybe we should clarify:

...To test the example plugins, copy ***or symbolically link*** the contents of...

Reply:
Thanks, fixed